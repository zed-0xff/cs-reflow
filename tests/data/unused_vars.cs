void keep_a1()
{
    int b;
    int a = b = 111;
    foo(a);
}

void keep_a2()
{
    int a;
    int b;
    a = b = 111;
    foo(a);
}

void keep_a3()
{
    int a, b;
    a = b = 111;
    foo(a);
}

void keep_b1()
{
    int b;
    int a = b = 111;
    foo(b);
}

void keep_b2()
{
    int a;
    int b;
    a = b = 111;
    foo(b);
}

void keep_b3()
{
    int a, b;
    a = b = 111;
    foo(b);
}

void keep_333()
{
    int b;
    foo(b = 333);
    b = 444;
}
