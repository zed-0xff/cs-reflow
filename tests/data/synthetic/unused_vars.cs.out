void keep_a1()
{
    int a = 111;
    foo(a);
}

void keep_a2()
{
    int a = 111;
    foo(a);
}

void keep_a3()
{
    int a = 111;
    foo(a);
}

void keep_a4()
{
    int a = 111;
    foo(a);
}

void keep_a5()
{
    int a = 111;
    foo(a);
}

void keep_b1()
{
    int b = 111;
    foo(b);
}

void keep_b2()
{
    int b = 111;
    foo(b);
}

void keep_b3()
{
    int b = 111;
    foo(b);
}

void keep_333()
{
    foo(333);
}

void block()
{
    if (foo())
    {
    }
}

